modules/simulations/k8s-managed/requirements.in

```diff
@@ -1,7 +1,7 @@
-aws-cdk-lib~=2.82.0
-aws-cdk.lambda-layer-kubectl-v29==2.0.0
```

Comment:
The module deployment was failing because version `2.0.0` was builddng a Python 3.7 lambda, which AWS no longer lets you do.